server/utils/helpers.js

```diff
@@ -0,0 +1,95 @@
+import axios from 'axios'
+import querystring from 'querystring'
+import consts from '../utils/consts'
+import jwtDecode from 'jwt-decode'
+
+export const decode = token => {
+  return token ? jwtDecode(token) : null
+}
+
+/**
+ * Handle possibility where token endpoint, at exp returns seconds instead of µ seconds
+ */
+export const handleTokenExp = exp => {
```

Comment:
This isn’t used here, but has been a solution when I lost a few hours in a day because my `exp` int value was seconds, instead of milliseconds. And it was failing silently.

Such a handler could be useful to ensure discrepancy between backend implementation and expectation.